### ISSUE TITLE
Unable to use watch on apps/admin

### ISSUE THREAD

**Issue by snstanton** [webiny/webiny-js#2219]
### Version

5.22.0

### Operating System

MacOS

### Browser

Firefox

### What steps will reproduce the bug?

`yarn webiny --env dev watch apps/admin`

When the app starts it will immediately overlay the UI with an error complaining about `Buffer` being undefined.  This happens outside of watch, too, but then it only logs to the console.

### What is the expected behavior?

There should be no error in the console or in the UI.

### Additional information

The error appears to be coming from @webiny/telemetry/sendEvent, which is using Buffer without importing it first.  I suspect the change in behavior is because webpack 5 no longer implicitly imports node.js packages.

**Comment by snstanton** [comment]
A workaround is to set 
```
REACT_APP_WEBINY_TELEMETRY=false
```

**Comment by Pavel910** [comment]
Thanks for reporting. On our side, telemetry is always disabled, so we never ran into this problem. 
Fixing it right away for 5.23.

**Comment by Pavel910** [comment]
This is fixed and `5.22.1` is released. You can just pull in the new npm packages. See the [Upgrade Guide](https://www.webiny.com/docs/release-notes/5.22.1/upgrade-guide).

Btw. to enable/disable telemetry, you don't need to set ENV variables yourself. There's a CLI command for that:
```
yarn webiny enable-telemetry
yarn webiny disable-telemetry
```
